triggers/daml/Daml/Trigger.daml

```diff
@@ -0,0 +1,51 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+daml 1.2
+module Daml.Trigger
```

Comment:
Leaving it at `Daml.Trigger` for now since that matches the fact that we call them DAML triggers and we might want to change other module names. We can make a decision on what we go with when we are closer to exposing users to this.